changelog/action.yml

```diff
@@ -22,6 +22,26 @@ runs:
       run: |
         curl -sSfL https://raw.githubusercontent.com/ory/ci/master/changelog/create_changelog.sh | bash
       shell: bash
+    - name: format the changelog
+      uses: ory/ci/prettier@master
+      with:
+        dir: .
+        action: write
+        target: CHANGELOG.md
+    # NOTE: need to format twice because of https://github.com/prettier/prettier/issues/13213
```

Comment:
The issue suggests that a correctly formatted md input would not cause this. Can we fix that during changelog generation @kevgo?

Reply:
I agree! Auto-generated code should also be formatted with Prettier because a lot of the advantages of consistent formatting apply also to auto-generated code. The best way to format auto-generated code is right when it is generated, so that we don't need to add extra formatting steps later, like had to be done here.

Reply:
> The issue suggests that a correctly formatted md input would not cause this. Can we fix that during changelog generation @kevgo?

@zepatrik That's how it was before this PR. 
The script that creates the changelog (`changelog/create_changelog.sh` in this repo) prettified the generated changelog right after creating it.

https://github.com/ory/ci/pull/108 restores this behavior by reverting most of the changes in this PR and fixing the problem that you discovered (missing Prettier dependencies). 

I'm moving all repos to managing dependencies via package.json. This has several advantages:
- all dependencies are properly versioned (via package-lock.json) --> no sudden surprises on CI 
- easier to secure the software supply chain against malicious dependencies by tools that scan package-lock.json
- less hacky, more straightforward to understand and maintain

Please connect with me if any of my changes  doesn't make sense to you. Thanks!